### ISSUE TITLE
Resized windows do not adjust their fields width both ways

### ISSUE THREAD

**Issue by RychuP** [vchelaru/FlatRedBall#1642]
I resized the Variables window and expected the fields to change their width to accommodate new window size and they did, but only one way: they got longer, never shorter. Expected behaviour: field length in a window resizes with the window both ways. Selecting a different item in the Explorer and returning to Variables makes the fields return to their normal size.

![windoresizingissue](https://github.com/user-attachments/assets/0f8d2ba4-e01b-4918-ae4c-3aa6ad04ab4a)

**Comment by vchelaru** [comment]
A bit of history on this bug. Initially, the size of a window depended on the contents of the "hint" text. This was causing very long scroll bars. To address this, I made a hacky width relationship between the hint text and the "top grid" which is where the main control was held. It works, in that it produces the results you see here.

I have no idea how to fully fix it and I spent a bit of time on it. I'll assign our resident WPF expert @vicdotexe on this, if he has time and if it's not too complicated maybe he can solve it.
